esx_service/volume_kv.py

```diff
@@ -58,6 +58,7 @@
 # The disk allocation format for vmdk
 DISK_ALLOCATION_FORMAT = 'diskformat'
 VALID_ALLOCATION_FORMATS = ["zeroedthick", "thin", "eagerzeroedthick"]
+CLONE_ADD_ALLOCATION_FORMAT = "delta" # Additional allocation format for clones
 DEFAULT_ALLOCATION_FORMAT = 'thin'
```

Comment:
I think it can be a source of confusion. And we already expose disk formats to the users.